mod/metrics/set/right/variables.rb

```diff
@@ -2,6 +2,11 @@
 include_set Abstract::Variable
 include_set Abstract::Table
 
+# FIXME: following does not prevent storage.
+event :abort_storage, :validate, on: :save do
+  abort :success
+end
```

Comment:
remove it if it doesn't do anything?

Reply:
commenting for now.  The story is still open on PT and we'll need to revisit it fairly soon.